syft/pkg/cataloger/cataloger.go

```diff
@@ -54,7 +54,8 @@ func ImageCatalogers(cfg Config) []Cataloger {
 		php.NewPHPComposerInstalledCataloger(),
 		javascript.NewJavascriptPackageCataloger(),
 		deb.NewDpkgdbCataloger(),
-		rpmdb.NewRpmdbCataloger(),
+		rpm.NewRpmdbCataloger(),
+		rpm.NewFileCataloger(),
```

Comment:
The image catalogers should only include catalogers that detect installed packages. The RPMDB is a good indicator of this, however, an RPM file in the image is not.